syntax = "proto3";

package sancus;

import "google/protobuf/any.proto";
import "google/protobuf/timestamp.proto";

message NextToken {
    google.protobuf.Any request = 1;
    google.protobuf.Timestamp initialRequestTimestamp = 2;
    uint64 offset = 3;
}
